firewall.go

```diff
@@ -35,6 +35,7 @@ type FirewallRule struct {
 	EndPort    string   `json:"end_port"`
 	Cidr       []string `json:"cidr"`
 	Direction  string   `json:"direction"`
+	Action     string   `json:"action"`
```

Comment:
Is action a required paramater? If not, what's the default on API side?

Reply:
Now yes, is required paramater, need to be send